src/core/components/courses/CourseCell/index.vue

```diff
@@ -134,10 +134,7 @@ export default {
 
     const formatNearestDate = computed(() => {
       if (!courseSlotsCount.value && course.value.estimatedStartDate) {
-        const rangeToEstimatedStartDate = getRoundedDiffInDays(
-          course.value.estimatedStartDate,
-          CompaniDate().startOf('day')
-        );
+        const rangeToEstimatedStartDate = Math.floor(CompaniDuration(course.value.durationToStartCourse).asDays());
```

Comment:
Pourquoi être passé de `Math.round` dans `getRoundedDiffInDays` à un `Math.floor` ici  ?
Que penses-tu de créer une méthode commune vu qu'on fait la meme chose pour tous les champs ? 
Est-ce qu'on ne pourrait pas directement calculer le champs ainsi dans `Trello` vu qu'on ne l'y utilise que pour le tri ?

Reply:
dans Trello on compare aujourd'hui (start of day) au moment du créneau, pour pouvoir trier deux créneaux le même jour. Dans courseCell on compare ajourd'hui (start of day) au jour du créneau (start of day) pour avoir un nombre de jours. Ca implique quelques modifications. Tous les créneaux en amont (formations terminées) doivent être pris à l'entier supérieur (si le créneau était hier, je veux que ça me renvoie "il y a 1 jour" et pas "il y a 0 jour" parce que ça ferait moins d'une demi journée) donc on utilise ceil et tous les créneaux en aval (formation en cours et à venir) doivenet être pris à l'entier inférieur (si le créneau est demain soir je veux que ça renvoie "dans 1 jour" et pas dans "2 jours" parce que ça ferait 1,6 jours) donc on utilise floor